docs/getting-started/installing-service-binding.md

```diff
@@ -6,11 +6,167 @@ sidebar_position: 1
 
 ## Compatiblity Note
 
-## Operator
+## Install on Kubernetes
 
-_How can To be provided soon_
+1. Install Operator Lifecycle Manager (OLM), a tool to help manage the Operators running on your cluster.
+
+```
+curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.18.1/install.sh | bash -s v0.18.1
```

Comment:
Would it be beneficial to show the output of this?

Reply:
There are different ways of installation - Manual, Scripted, Compile from master.

I have provided the link to Installation Guide in the doc and removed the command.